docs/UserManuals/ConfigUI/GitLab.md

```diff
@@ -27,24 +27,22 @@ If you are behind a corporate firewall or VPN you may need to utilize a proxy se
 #### Test and Save Connection
 Click `Test Connection`, if the connection is successful, click `Save Connection` to add the connection.
 
-
 ### Step 2 - Setting Data Scope
 
-#### Projects
-Enter the GitLab repos to collect. How to get `GitLab` repos?
-- Visit the repository page on GitLab
-- Find the project id below the title
+![image](https://user-images.githubusercontent.com/3294100/199533780-f506b308-6808-499c-90db-b39fcda27888.png)
 
-![Get GitLab projects](https://user-images.githubusercontent.com/3789273/128568416-a47b2763-51d8-4a6a-8a8b-396512bffb03.png)
+#### Projects
+Choose the Gitlab projects to collect. Limited by GitLab API, You need to type more than 2 characters to search.
 
-If you want to collect more than 1 repo, please separate repos with comma. For example, "apache/incubator-devlake,apache/incubator-devlake-website".
+* If you want to collect public repo in GitLab, please uncheck "Only search my repositories" to find all projects.
```

Comment:
Text Change:
“If you want to collect public repos in GitLab, please uncheck "Only search my repositories" to find all public repos.”